pynucastro/networks/tests/test_cxx_amrexastro_net.py

```diff
@@ -80,8 +80,8 @@ def test_write_network(self, fn):
         reference_path = "_amrexastro_cxx_reference/"
         # files that will be ignored if present in the generated directory
         skip_files = [
-            "23Na-23Ne_electroncapture.dat",
-            "23Ne-23Na_betadecay.dat",
+            "23na-23ne_electroncapture.dat",
+            "23ne-23na_betadecay.dat",
         ]
```

Comment:
These files should either be deleted from the reference directory, or removed from `skip_files` so they get checked, as they're currently out of date.